sktime/forecasting/varmax.py

```diff
@@ -0,0 +1,299 @@
+__author__ = ["KatieBuc"]
+
+import numpy as np
+import pandas as pd
+from statsmodels.tsa.statespace.varmax import VARMAX as _VARMAX
+
+from sktime.forecasting.base.adapters import _StatsModelsAdapter
+
+
+class VARMAX(_StatsModelsAdapter):
+    """
+    Wrapper for statsmodels VARMAX model
+
+    Vector Autoregressive Moving Average with eXogenous regressors model (VARMAX)
+
+    Parameters
+    ----------
+    y : array_like
+        The observed time-series process :math:`y`, , shaped n_obs x k_endog.
+    X : array_like, optional
+        Array of exogenous regressors, shaped n_obs x k.
+    order : iterable
+        The (p,q) order of the model for the number of AR and MA parameters to
+        use.
+    trend : str{'n','c','t','ct'} or iterable, optional
+        Parameter controlling the deterministic trend polynomial :math:`A(t)`.
+        Can be specified as a string where 'c' indicates a constant (i.e. a
+        degree zero component of the trend polynomial), 't' indicates a
+        linear trend with time, and 'ct' is both. Can also be specified as an
+        iterable defining the non-zero polynomial exponents to include, in
+        increasing order. For example, `[1,1,0,1]` denotes
+        :math:`a + bt + ct^3`. Default is a constant trend component.
+    error_cov_type : {'diagonal', 'unstructured'}, optional
+        The structure of the covariance matrix of the error term, where
+        "unstructured" puts no restrictions on the matrix and "diagonal"
+        requires it to be a diagonal matrix (uncorrelated errors). Default is
+        "unstructured".
+    measurement_error : bool, optional
+        Whether or not to assume the endogenous observations `endog` were
+        measured with error. Default is False.
+    enforce_stationarity : bool, optional
+        Whether or not to transform the AR parameters to enforce stationarity
+        in the autoregressive component of the model. Default is True.
+    enforce_invertibility : bool, optional
+        Whether or not to transform the MA parameters to enforce invertibility
+        in the moving average component of the model. Default is True.
+    trend_offset : int, optional
+        The offset at which to start time trend values. Default is 1, so that
+        if `trend='t'` the trend is equal to 1, 2, ..., n_obs. Typically is only
+        set when the model created by extending a previous dataset.
+    **kwargs
+        Keyword arguments may be used to provide default values for state space
+        matrices or for Kalman filtering options. See `Representation`, and
+        `KalmanFilter` for more details.
+    Attributes
+    ----------
+    order : iterable
+        The (p,q) order of the model for the number of AR and MA parameters to
+        use.
+    trend : str{'n','c','t','ct'} or iterable
+        Parameter controlling the deterministic trend polynomial :math:`A(t)`.
+        Can be specified as a string where 'c' indicates a constant (i.e. a
```

Comment:
This attributes section is a copy of (a subset of) the parameters section. I think it is safe to just delete this section, as it does not provide any additional information for a user.